[package] bullet3/3.25: "Bullet::Bullet" includes non-existent path

Headers belonging to extras come inside the `include/bullet` directory like everything else. However, the recipe  adds `include/bullet_robotics` to the include dirs. This is probably outdated, because it causes the following error:
```
CMake Error in CMakeLists.txt:
  Imported target "Bullet::Bullet" includes non-existent path
    "/Users/userhome/development/explore/conan/p/b/bulle2a3d4020848c9/p/include/bullet_robotics"
```

--- a/recipes/bullet3/all/conandata.yml
+++ b/recipes/bullet3/all/conandata.yml
@@ -8,6 +8,3 @@ sources:
   "3.22a":
     url: "https://github.com/bulletphysics/bullet3/archive/refs/tags/3.22a.tar.gz"
     sha256: "c40a0e832751c1590474cf21679972d2938e0705a45c29fed846ddb523d82b03"
-  "2.89": #remove next time a new version is added
-    url: "https://github.com/bulletphysics/bullet3/archive/2.89.tar.gz"
-    sha256: "621b36e91c0371933f3c2156db22c083383164881d2a6b84636759dc4cbb0bb8"

--- a/recipes/bullet3/all/conanfile.py
+++ b/recipes/bullet3/all/conanfile.py
@@ -78,9 +78,6 @@ class Bullet3Conan(ConanFile):
         tc.variables["BUILD_UNIT_TESTS"] = False
         if is_msvc(self):
             tc.variables["USE_MSVC_RUNTIME_LIBRARY_DLL"] = not is_msvc_static_runtime(self)
-        if Version(self.version) < "3.21":
-            # silence warning
-            tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0115"] = "OLD"
         if Version(self.version) <= "3.25":
             tc.cache_variables["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5"  # CMake 4 support
         tc.generate()

--- a/recipes/bullet3/all/conanfile.py
+++ b/recipes/bullet3/all/conanfile.py
@@ -180,8 +180,6 @@ class Bullet3Conan(ConanFile):
 
         self.cpp_info.libs = libs
         self.cpp_info.includedirs = ["include", os.path.join("include", "bullet")]
-        if self.options.extras:
-            self.cpp_info.includedirs.append(os.path.join("include", "bullet_robotics"))
         self.cpp_info.defines = self._bullet_definitions
         if self.options.bt2_thread_locks and self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.system_libs.append("pthread")

--- a/recipes/bullet3/config.yml
+++ b/recipes/bullet3/config.yml
@@ -5,5 +5,4 @@ versions:
     folder: all
   "3.22a":
     folder: all
-  "2.89": #remove next time a new version is added
-    folder: all
+

